Cargo.toml

```diff
@@ -71,18 +71,22 @@ version = "0.7"
 name = "demo"
 path = "examples/demo/demo.rs"
 
+[[example]]
+name = "egui"
+path = "examples/egui/egui.rs"
+
 [[example]]
 name = "fox"
 path = "examples/animation/fox.rs"
 
+[[example]]
+name = "light"
+path = "examples/light/light.rs"
+
 [[example]]
 name = "skybox"
 path = "examples/skybox/skybox.rs"
 
 [[example]]
 name = "ui"
```

Comment:
[question]
This example is empty, right? Maybe it is better to remove it for now from main. What do you think?

Reply:
Yes, it is empty. We don't need it now. I will remove it